cmd/init.go

```diff
@@ -43,6 +43,12 @@ func init() {
 	initCmd.PersistentFlags().StringVar(&etcdAdmConfig.ReleaseURL, "release-url", constants.DefaultReleaseURL, "URL used to download etcd")
 	initCmd.PersistentFlags().StringVar(&etcdAdmConfig.CertificatesDir, "certs-dir", constants.DefaultCertificateDir, "certificates directory")
 	initCmd.PersistentFlags().StringVar(&etcdAdmConfig.BindAddr, "bind-address", "", "etcd bind address")
+	initCmd.PersistentFlags().StringVar(&etcdAdmConfig.UnitFile, "unit-file", "/etc/systemd/system/etcd.service", "system unit file")
```

Comment:
So IMO we should use either `constants.Foo` or the etcdAdmConfig value (which would have been set in SetDefaults).  It looks like we're doing the first form, though I personally prefer the second form.  The second form would be e.g.

`initCmd.PersistentFlags().IntVar(&etcdAdmConfig.PeerPort, "peer-port", etcdAdmConfig.PeerPort, "etcd port listening from peer etcd instance")`